### ISSUE TITLE
Windows 10 64bit: Error when configuring vst3sdk

### ISSUE THREAD

**Issue by COx2** [hotwatermorning/Terra#19]
Platform: Windows 10 64bit
I ran the following command:

```
cd ./gradle
./gradlew build_all -Pconfig=Debug # or Release
open ../build_debug/Debug/Terra.app
```

However, Cmake configure fails with the following error message
```
[cmd, /c, chcp, 65001, >, nul, &, cmake, -G, Visual Studio 15 2017, -Ax64, -Tv141,host=x64, -DCMAKE_INSTALL_PREFIX=C:\Users\ShiozawaTatsuya\Documents\GitRepos\Terra\ext\vst3sdk\build_debug/install, -DSMTG_ADD_VST3_PLUGINS_SAMPLES=OFF, -DSMTG_ADD_VST3_HOSTING_SAMPLES=OFF, -DSMTG_ADD_VSTGUI=OFF, ..] @ 'C:\Users\ShiozawaTatsuya\Documents\GitRepos\Terra\ext\vst3sdk\build_debug' ([CC:/c/_tools/ccache-win64/ccache clang, CXX:/c/_tools/ccache-win64/ccache clang++])
CMake Error at cmake/modules/AddVST3Options.cmake:47 (file):
-- SMTG_VST3_TARGET_PATH is not set!
-- SMTG_MYPLUGINS_SRC_PATH is not set. If you want to add your own plug-ins folder, specify it!
  file problem creating directory: C:/Program Files (x86)/Common Files/VST3
Call Stack (most recent call first):
  CMakeLists.txt:21 (include)
```

The problem is easy to solve, just create a folder "C:/Program Files (x86)/Common Files/VST3" with using File Explorer.
But this is a strange behavior. Because the vst3sdk Cmake script is written to check if there is a folder "C:/Program Files/Common Files/VST3" for x64 builds.

https://github.com/steinbergmedia/vst3_cmake/blob/2839d67bce24c75371cbb162f0a377eef6ef665c/modules/SetupVST3LibraryDefaultPath.cmake#L8

**Comment by hotwatermorning** [comment]
Recently I began to use `-A` option of cmake command since this commit: 1109f1ef to specify target platform, but it seems that vst3sdk does not support this option and only supports old style generator string for detecting `DEFAULT_VST3_FOLDER`.

I'll fix build.gradle to use old style generator string.

**Comment by hotwatermorning** [comment]
Fixed in f9d17f9ed6940201dead33b3528a8a05ad097567.

Thanks @COx2

**Comment by COx2** [comment]
Great!!
